pkgs/development/python-modules/seabreeze/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+## Usage
+# In NixOS, simply add the `udev` multiple output to services.udev.packages:
+#   services.udev.packages = [ pkgs.python3Packages.seabreeze.udev ];
+
+python3Packages.buildPythonPackage rec {
+  pname = "seabreeze";
+  version = "v60569e";
```

Comment:
Noticing that this isn't a tagged release but a commit from master.
See https://nixos.org/nixpkgs/manual/#sec-package-naming
and specifically "If a package is not a release but a commit from a repository"